app/views/podcast_episodes/index.html.erb

```diff
@@ -2,18 +2,20 @@
   <%= render "podcast_episodes/meta" %>
 <% end %>
 <% if @podcast %>
-  <% image = Images::Optimizer.call(@podcast.pattern_image_url || "https://i.imgur.com/fKYKgo4.png", fetch_format: "jpg", crop: nil) %>
-  <div class="user-profile-header podcast-header" style="background:#<%= @podcast.main_color_hex %> url(<%= image %>);">
+  <div class="user-profile-header podcast-header" style="background:#<%= @podcast.main_color_hex %>;">
     <div class="tag-or-query-header-container">
       <h1><img class="record main-image" src="<%= Images::Optimizer.call(@podcast.image_url, crop: "fill", width: 420, height: 420) %>"
                alt="<%= @podcast.title %>" /> <%= @podcast.title %>
-               <button
-                id="user-follow-butt"
-                class="crayons-btn follow-action-button user-profile-follow-button"
-                data-info='{"id":<%= @podcast.id %>,"className":"<%= @podcast.class.name %>", "name": "<%= @podcast.name %>"}'>
-                  &nbsp;
-              </button>
-</h1>
+      </h1>
+      <h2>
+        <button
+          id="user-follow-butt"
+          class="crayons-btn follow-action-button"
+          style="min-width: 100px;"
+          data-info='{"id":<%= @podcast.id %>,"className":"<%= @podcast.class.name %>", "name": "<%= @podcast.name %>"}'>
+            &nbsp;
+        </button>
+      </h2>
```

Comment:
```suggestion
        <button
          id="user-follow-butt"
          class="crayons-btn follow-action-button"
          style="min-width: 100px;"
          data-info='{"id":<%= @podcast.id %>,"className":"<%= @podcast.class.name %>", "name": "<%= @podcast.name %>"}'>
            &nbsp;
        </button>
```

suggestion (not blocking) - we should change this to remove the heading, since "Follow" doesn't really represent a section of the document, it's just a button on its own. It wasn't great semantic HTML before to have it inside the `h1`, so happy to leave as a non-blocking suggestion we can fix now or later